src/base/bittorrent/session.cpp

```diff
@@ -155,6 +155,19 @@ namespace
 
         return expanded;
     }
+
+    QStringList findAllFiles(const QString &folderPath)
+    {
+        QStringList files;
```

Comment:
@evsh:  True, but generally speaking it's not the program's job to handle such a case.  The operating system usually tells the program that the directory is either invalid or lacking correct permissions.  

Adding code to qBittorrent to handle symbolic links would be both a huge undertaking (program-wide and different inplementations to support every OS / filesystem combination), and provide very little benefit overall, since the entire point of symbolic links is they're supposed to be invisible to the programs that access them.